src/applications/caregivers/components/AdditionalInfo/index.jsx

```diff
@@ -0,0 +1,124 @@
+      <a
+        href="https://www.law.cornell.edu/cfr/text/38/71.25"
+        target="_blank"
+        rel="noopener noreferrer"
+      >
+        38 C.F.R. §71.25.
+      </a>
+    </p>
+    <a
+      href="https://www.va.gov/healthbenefits/resources/Caregiver_Eligibility_Check.asp"
+      target="_blank"
+      rel="noopener noreferrer"
+    >
+      Find out if you qualify
+    </a>
+  </AdditionalInfo>
+);
+
+export const SecondaryCaregiverInfo = () => (
+  <AdditionalInfo triggerText="Learn more about Secondary Caregiver">
+    <p>
+      An individual approved as a "provider of personal care services" for the
+      eligible Veteran under{' '}
+      <a
+        href="https://www.law.cornell.edu/uscode/text/38/1720G"
+        target="_blank"
+        rel="noopener noreferrer"
+      >
+        38 U.S.C. §1720G(a)(7)(A);
+      </a>
+      meets the requirements of{' '}
+      <a
+        href="https://www.law.cornell.edu/cfr/text/38/71.25"
+        target="_blank"
+        rel="noopener noreferrer"
+      >
+        38 C.F.R. §71.25;
+      </a>
+      and generally serves as a back-up to the Primary Family Caregiver.
+    </p>
+    <a
+      href="https://www.va.gov/healthbenefits/resources/Caregiver_Eligibility_Check.asp"
+      target="_blank"
+      rel="noopener noreferrer"
+    >
+      Find out if you qualify
+    </a>
+  </AdditionalInfo>
+);
+
+export const CHAMPVAInfo = () => (
+  <AdditionalInfo triggerText="Learn more about CHAMPVA">
+    <p>
+      CHAMPVA is a cost-sharing program that covers the price of some health
+      care services and supplies.
+    </p>
+    <a href="" target="_blank" rel="noopener noreferrer">
+      Learn more about CHAMPVA
+    </a>
+  </AdditionalInfo>
+);
+
+export const CaregiverSupportInfo = () => (
+  <AdditionalInfo triggerText="Learn more about Caregiver Support Coordinator">
+    Caregiver Support Coordinators (CSC) connect caregivers with VA and
+    community resources. They are located at every VA medical center and
+    specialize in caregiving issues.
+    <a href="" target="_blank" rel="noopener noreferrer">
+      Find a caregiver support coordinator near you.
+    </a>
+  </AdditionalInfo>
+);
+
+export const PowerOfAttorneyInfo = () => (
+  <AdditionalInfo triggerText="Learn more about Power of Attorney">
+    Power of attorney authorizes someone to act on the behalf of a Veteran or
+    service member when completing this form.
+  </AdditionalInfo>
+);
+export const RepresentativeInfo = () => (
+  <AdditionalInfo triggerText="Learn more about Representative">
+    Refers to a Veteran's or Servicemember's court-appointed legal guardian or
+    special guardian Durable POA for Health Care, or other designated health
+    care agent. Attach POA/Representation documents to the application if
+    applicable.
+    <span>You can also:</span>
+    <ol>
+      <li>Call us at 877-222-VETS (877-222-8387)</li>
+      <li>
+        Find a <a href="">Caregiver Support Coordinator</a>
```

Comment:
Link needs `href` 🙂Just going to note all of them so you can reference them later!

Reply:
Did not have link will ask designer for them

Reply:
Fixed!

Reply:
In cases like this, I like to add the keyword 'TODO' or something like that so I can easily search for all my missed bookmarks like this. Otherwise, you'd have to search for 'href=""' or something tedious like that.